src/mqt/predictor/ml/Predictor.py

```diff
@@ -413,7 +433,7 @@ def plot_eval_histogram(
         plt.savefig(result_path / (filename + ".pdf"))
         plt.show()
 
-        return res
+        return
```

Comment:
that return statement should be superfluous if I am not mistaken

Reply:
Removed it.